internal/slicer/slicer.go

```diff
@@ -246,6 +250,17 @@ func Run(options *RunOptions) error {
 		}
 	}
 
+	for _, path := range implicitConflicts {
+		// A directory is listed in the report if and only if it was listed
+		// explictly in an SDF, meaning there is no conflict.
```

Comment:
s/an SDF/the slice contents/

The logic is also also not complete, in the sense that it's not that there might be no conflict, but rather that we detect such conflicts earlier as we forbid extracted content from multiple packages when paths match.